conans/test/unittests/tools/files_patch_test.py

```diff
@@ -184,11 +184,90 @@ def build(self):
         client.save({"conanfile.py": file_content})
         client.run("install .")
         client.run("build .", assert_error=True)
-        self.assertIn("patch: error: no patch data found!", client.out)
+        self.assertIn("patch_ng: error: no patch data found!", client.out)
         self.assertIn("ERROR: conanfile.py (test/1.9.10): "
                       "Error in build() method, line 12", client.out)
         self.assertIn("Failed to parse patch: string", client.out)
 
+    def test_add_new_file(self):
```

Comment:
If we are not testing how to build something, we try to omit call CMake because the tests are quite slow.
Replace the test with a patch to adding a file, and do an assert inside the `build()` method. Simpler and faster.

Reply:
done